### PR TITLE
enhance(chart_revision): add summary text, large viz option

### PR DESCRIPTION
**Pull request by lucasrodes**
For the chart revision tool I am working on surfacing details on what is changing in the chart. To this end, I have added a new field `changesInDataSummary` to our database table `suggested_chart_revisions`.

Now we need to allocate some space on the grapher side to surface this.

This PR:

- Adds HTML field to surface the value of `suggested_chart_revisions`.
- Additionally, I am adding the option to have preview size `"Large"`, too.
- Change heading tags: `<h3>` → `<h1>` and `<h5>` → `<h2>`.

Parallel work on the ETL-side: https://github.com/owid/etl/pull/987

### CODE REVIEW

File: adminSiteClient/SuggestedChartRevisionApproverPage.tsx [hunk 1]
```diff
@@ -348,7 +350,7 @@ export class SuggestedChartRevisionApproverPage extends React.Component<{
                 noSidebar
             >
                 <main className="SuggestedChartRevisionApproverPage">
-                    <h3>
+                    <h1>
```
Comment (marcelgerber) [inline adminSiteClient/SuggestedChartRevisionApproverPage.tsx:353]: All these heading level changes seem a bit odd to me, they look a little too prominent now.
Not sure if these were all intentional?

File: adminSiteClient/SuggestedChartRevisionApproverPage.tsx [hunk 2]
```diff
@@ -1206,9 +1210,23 @@ export class SuggestedChartRevisionApproverPage extends React.Component<{
...
+                <div className="changes_summary">
+                    <h2>Variable changes</h2>{" "}
+                    {this.suggestedChartRevision &&
+                    this.suggestedChartRevision.changesInDataSummary ? (
+                        <div
+                            dangerouslySetInnerHTML={{
+                                __html: this.suggestedChartRevision
+                                    .changesInDataSummary,
+                            }}
+                        ></div>
```
Comment (marcelgerber) [inline adminSiteClient/SuggestedChartRevisionApproverPage.tsx:1225]:
```suggestion
                        <div>
                            {this.suggestedChartRevision.changesInDataSummary}
                        </div>
```

**Review by marcelgerber — COMMENTED** [review]
Nice, looks mostly good.

Please beware that you will have to remove the column from the live db again (if you didn't do that already), otherwise the migration will fail to run.

File: Makefile [hunk 3]
```diff
@@ -125,7 +125,7 @@ up.full: require create-if-missing.env.full wordpress/.env tmp-downloads/owid_ch
 
 migrate:
 	@echo '==> Running DB migrations'
-	yarn && yarn buildTsc && yarn runDbMigrations
+	@. ./.env && yarn && yarn buildTsc && yarn runDbMigrations
```
[the remark this answers is not in the document]
Comment (lucasrodes) [reply Makefile:128]: cc. @Marigold

File: adminSiteClient/SuggestedChartRevisionApproverPage.tsx (hunk 2 above)
Comment (lucasrodes) [reply adminSiteClient/SuggestedChartRevisionApproverPage.tsx:1225]: This appears not to render correctly. The thing is, `this.suggestedChartRevision.changesInDataSummary` is HTML-rich.

Comment (marcelgerber) [reply adminSiteClient/SuggestedChartRevisionApproverPage.tsx:1226]: ah, okay, yeah then that's correct. I just didn't expect it to be HTML.

File: adminSiteClient/SuggestedChartRevisionApproverPage.tsx (hunk 1 above)
Comment (lucasrodes) [reply adminSiteClient/SuggestedChartRevisionApproverPage.tsx:353]: I changed them a bit, I think it looks fine.

I left to H1, the main title, and the one for the actual revision

File: Makefile (hunk 3 above)
[the remark this answers is not in the document]
Comment (Marigold) [reply Makefile:128]: It didn't load .env variables for me, but feel free to remove it from this PR. I'll look into it on my local.

[the remark this answers is not in the document]
Comment (Marigold) [reply Makefile:128]: My bad, I had "GRAPHER_DB_PASS=''" in my `.env` and `include .env` was loading it as "''" while `./.env` was interpreting it correctly.

**Review by marcelgerber — APPROVED** [review]

File: Makefile (hunk 3 above)
[the remark this answers is not in the document]
Comment (lucasrodes) [reply Makefile]: I'm rolling back to the previous state then, removing `@. ./.env && `

**Pull request merged**